tracer/src/Datadog.AutoInstrumentation.NativeLoader/cor_profiler.cpp

```diff
@@ -189,12 +193,26 @@ namespace datadog::shared::nativeloader
         Log::Debug("CorProfiler::Initialize: MaskLow: ", mask_low);
         Log::Debug("CorProfiler::Initialize: MaskHi : ", mask_hi);
 
+        if (instrumented_assembly_generator::IsInstrumentedAssemblyGeneratorEnabled())
+        {
+            m_writeToDiskCorProfilerInfo = new instrumented_assembly_generator::InstrumentedAssemblyGeneratorCorProfilerInfo(pICorProfilerInfoUnk);
+            isWriteInstrumentationToDiskEnabled = true;
```

Comment:
`isWriteInstrumentationToDiskEnabled` is not really needed. You can check if `m_writeToDiskCorProfilerInfo` pointer is equal to `nullptr` instead

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a